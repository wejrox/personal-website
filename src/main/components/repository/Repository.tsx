import * as React from "react";
import Row from "reactstrap/lib/Row";
import Col from "reactstrap/lib/Col";

interface RepositoryProps {

}

class Repository extends React.Component<RepositoryProps> {

    public render() {
        return (
            <Row>
                <Col>
                    Gallery
                </Col>
            </Row>
        );
    }
}

export default Repository;