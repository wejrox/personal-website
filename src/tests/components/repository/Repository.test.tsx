import * as Enzyme from "enzyme";
import * as React from "react";
import Repository from "../../../main/components/repository/Repository";

describe("Repository Component", () => {
    const wrapper = Enzyme.shallow(<Repository/>);

    it("renders correctly", () => {
        expect(wrapper).toMatchSnapshot();
    });
});