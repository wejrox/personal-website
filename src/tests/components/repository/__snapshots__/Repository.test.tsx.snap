// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repository Component renders correctly 1`] = `ShallowWrapper {}`;
